packit.spec

```diff
@@ -31,6 +31,8 @@ BuildRequires:  python3dist(setuptools-scm-git-archive)
 BuildRequires:  python3-bodhi-client
 # new-sources
 Requires:       fedpkg
+# Copying files between repositories
+Requires:       rsync
```

Comment:
Is this the right place to put it, or should it be `BuildRequires`? This is needed to run the tests, but I'm not 100% if those are included in the build?

Reply:
If the RPM fails to build (by packit-service or manually with mock) then it's needed also in `BuildRequires`, otherwise probably not :-)